### PR TITLE
Fix the PSpellEngine getSuggestions function

### PR DESCRIPTION
**Pull request by segrax**
The getSuggestions function wasn't returning any suggestions, only which words had been misspelt

### CODE REVIEW

File: includes/PSpellEngine.php [hunk 1]
```diff
@@ -49,7 +49,11 @@ public function getSuggestions($lang, $words) {
 		$outWords = array();
 		foreach ($words as $word) {
 			if (!pspell_check($plink, trim($word))) {
-				$outWords[] = utf8_encode($word);
+				$outWords[$word] = array();
+
+				foreach( pspell_suggest($plink, trim($word)) as $suggestedWord ) {
+					$outWords[$word][] = utf8_encode($suggestedWord);
```
Comment (mvanbaak) [inline includes/PSpellEngine.php:55]: If pspell already outputs utf-8 things will be double encoded, resulting in broken output

Comment (segrax) [inline includes/PSpellEngine.php:55]: Thanks, i'll look into a proper fix for it soon

Comment (mvanbaak) [inline includes/PSpellEngine.php:55]: see #19

**Comment by tinydylan** [comment]
This GitHub repository contains the server-side code for the TinyMCE Community SpellChecker Plugin. This plugin was deprecated with the release of TinyMCE 5.4. No more development will be done on this plugin, and we are closing all associated pull requests.

For more information, please see:

[TinyMCE Spellchecker: What are your Options](https://www.tiny.cloud/blog/tinymce-spellchecker)

[TinyMCE 5.4 Release Notes](https://www.tiny.cloud/docs/release-notes/release-notes54/#thefreetinymcespellcheckerplugin)

**Pull request closed without merging**